index.html

```diff
@@ -2595,6 +2595,133 @@ <h3>Data Schemas</h3>
 
       </section>
 
+      <section>
+        <h2>Integrity of Related Resources</h2>
+        <p>
+          When including a link to an external resource in a
+          <a>verifiable credential</a>, it is desirable to know whether
+          the resource that is pointed to is the same at signing time as
+          it is at verification time.  This applies to cases where there
+          is an external resource that is remotely retrieved as well as
+          to cases where the <a>issuer</a> and/or
+          <a>verifier</a> may have local cached copies of a resource.
+        </p>
+        <p>
+          It is also desirable to know that the contents of the JSON-LD
+          context(s) used in the <a>verifiable credential</a> are the
+          same when used by both the <a>issuer</a> and <a>verifier</a>. 
+        </p>
+        <p>
+          To validate that a resource referenced by a <a>verifiable
+          credential</a> is the same at verification time as it is at
+          issuing time, an implementer MAY include a property named
+          <code>relatedResource</code> that stores an array of objects
+          that describe additional integrity metadata about each
+          resource referenced by the <a>verifiable credential</a>. If
+          <code>relatedResource</code>
+          is present, there MUST be an object in the array for each remote
+          resource for each context used in the verifiable credential. 
```

Comment:
I read this many times, and wasn't sure wether this was the intention. 
```suggestion
          is present, there MUST be an object in the array for each remote
          resource and for each context used in the verifiable credential. 
```